src/main/java/edu/illinois/cs/dt/tools/minimizer/MinimizerPlugin.java

```diff
@@ -59,19 +59,45 @@ public MinimizerPlugin(final InstrumentingSmartRunner runner) {
         TestPluginPlugin.info("Creating minimizers for file: " + path);
 
         try {
+            Path originalOrderPath = DetectorPathManager.originalOrderPath();
             List<String> originalOrder;
+            if (Files.exists(originalOrderPath)) {
+                originalOrder = Files.readAllLines(originalOrderPath);
+            } else {
+                originalOrder = new ArrayList<>();
+            }
+
             if (ORIGINAL_ORDER != null) {
                 TestPluginPlugin.info("Using specified original order. dt.minimizer.original.order argument specified: " + ORIGINAL_ORDER);
-                originalOrder = Files.readAllLines(Paths.get(ORIGINAL_ORDER));
-            } else {
-                originalOrder = DetectorPlugin.getOriginalOrder(project);
+                List<String> originalOrderSpecified = Files.readAllLines(Paths.get(ORIGINAL_ORDER));
+
+                if (originalOrderSpecified.equals(originalOrder)) {
+                    TestPluginPlugin.info("Custom original order specified and found to be matching the existing original order at: "
+                                          + originalOrderPath);
+                } else {
+                    TestPluginPlugin.info("Copying custom original order specified since it is different than the (non)existing original order at: "
+                                          + originalOrderPath);
+
+                    if (Files.exists(originalOrderPath)) {
+                        String originalOrderRename = "original-order-" + System.currentTimeMillis();
+                        Files.move(originalOrderPath, originalOrderPath.resolveSibling(originalOrderRename));
+                        TestPluginPlugin.info("Original order before copying is now moved to: " + originalOrderRename);
```

Comment:
Instead of saying originalOrderRename, which is just the base name, do you want to resolve it to the full path name?